src/Promitor.Core.Scraping/Configuration/Model/MetricsDeclaration.cs

```diff
@@ -5,7 +5,6 @@ namespace Promitor.Core.Scraping.Configuration.Model
 {
     public class MetricsDeclaration
     {
-        public string Version { get; set; }
```

Comment:
Yeah - it wouldn't have caused any issue to leave it, but I figured that it didn't really make sense since the internal model doesn't have a version - it's just the configuration that is versioned.